spec/requests/titles_spec.rb

```diff
@@ -86,6 +86,119 @@
         expect(json_f.errors.first.title).to eql('Invalid filter parameter')
       end
     end
+
+    describe 'with search field titlename filter' do
+      before do
+        VCR.use_cassette('search-titles-filter-titlename') do
+          get '/eholdings/titles/?filter[name]=ebsco', headers: okapi_headers
+        end
+      end
+
+      let!(:json_t) { Map JSON.parse response.body }
+
+      it 'gets a filtered list of resources' do
+        expect(response).to have_http_status(200)
+        expect(json_t.data.length).to equal(25)
+        expect(json_t.meta.totalResults).to equal(61)
+        json_t.data.each do |title|
+          expect(title.attributes.name.downcase).to include('ebsco')
+        end
+      end
+
+      describe 'with titlename filter and pagination' do
+        before do
+          VCR.use_cassette('search-titles-filter-titlename-page2') do
+            get '/eholdings/titles/?filter[name]=ebsco&page=2',
+                headers: okapi_headers
+          end
+        end
+        let!(:json_t2) { Map JSON.parse response.body }
+        it 'gets a different list of resources' do
+          expect(response).to have_http_status(200)
+          expect(json_t2.data.length).to equal(25)
+          expect(json_t2.meta.totalResults).to equal(61)
+          expect(json_t2.data.first.id).not_to eql(json_t.data.first.id)
+        end
+      end
+    end
+    describe 'with search field isxn filter' do
```

Comment:
I'm just being nitpicky now, but could we add an empty line before this like the other `describe` blocks?